task/prefetch-dependencies-oci-ta/0.1/prefetch-dependencies-oci-ta.yaml

```diff
@@ -325,10 +183,58 @@ spec:
       script: |
         #!/bin/bash
 
-        if [ -f /shared/skip ]; then
-          echo "Skipping."
-          exit 0
-        fi
+        RHSM_ORG=""
+        RHSM_ACT_KEY=""
+        ENTITLEMENT_CERT_PATH=""
+        ENTITLEMENT_CERT_KEY_PATH=""
+
+        function rhsm_unregister {
+          if [ -e /activation-key/org ]; then
+            # best effort:
+            #   - if the system was already successfully unregistered, the command returns 1
+            #   - if unregistering failed/fails, there's not much we can do about it anyway
+            subscription-manager unregister || true
+          fi
+        }
+
+        function inject_ssl_opts {
+          input="$1"
+          ssl_options="$2"
+
+          # Check if input is plain string or JSON and if the request specifies RPMs
+          if [ "$input" == "rpm" ]; then
+            input="$(
+              jq -n --argjson ssl_options "$ssl_options" '
+                      {
+                        type: "rpm",
+                        options: {
+                          ssl: $ssl_options
+                        }
+                      }'
+            )"
+          elif (jq -e . 2>/dev/null 1>&2 <<<"$input"); then
+            input_json_has_rpm="$(jq -e '
```

Comment:
Careful with `jq -e` - in combination with `set -e`, it will cause the script to exit if the expression evaluates to something falsy